tsconfig.tmp.json

```diff
@@ -8,6 +8,9 @@
       "@module-federation/sdk/normalize-webpack-path": [
         "dist/packages/sdk/normalize-webpack-path.cjs.js"
       ],
+      "@module-federation/webpack-bundler-runtime": ["dist/packages/webpack-bundler-runtime/index.cjs.js"],
+      "@module-federation/runtime": ["dist/packages/runtime"],
+      "@module-federation/runtime/*": ["dist/packages/runtime/*"],
```

Comment:
RELEASE_NUMBER will throw undefined error un upper plugin who uses tsconfig to resolve